src/components/Card.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import styled from 'styled-components';
+import { colors, device} from '../Theme';
+import NotFound from '../resources/noimage.png';
+import { useNavigate } from "react-router-dom";
+
+const Container = styled.div`
+    width: 100%;
+    height: 310px;
```

Comment:
Considering the amount of text that could come from these videos, I would suggest one of two solutions:

- remove the height completely or change it to a minimum height. This way, your content can freely grow to always display content without cropping it.
- You can leave the height fixed, but you need to verify how much text will appear on your card. An example would be to check how much height one line of title and one line of description would take, then adjust the height of the element to always display one of each and adjust the text to display ellipsis at the end of the line (text with dots at the end: example: "Welcome to the fantas...")